docs/reference/monitoring/configuring-metricbeat.asciidoc

```diff
@@ -42,8 +42,16 @@ view the cluster settings and `manage` cluster privileges to change them.
 For more information, see <<monitoring-settings>> and <<cluster-update-settings>>.
 --
 
-. {metricbeat-ref}/metricbeat-installation-configuration.html[Install {metricbeat}] on each
-{es} node in the production cluster. Failure to install on each node may result in incomplete or missing results.
+. {metricbeat-ref}/metricbeat-installation-configuration.html[Install
+{metricbeat}]. Ideally install a single {metricbeat} instance configured with
+`scope: cluster` and configure `hosts` to point to an endpoint (e.g. a
+load-balancing proxy) which directs requests to the master-ineligible nodes in
+the cluster. If this is not possible, and the cluster does not use dedicated
+master-eligible nodes, then install one {metricbeat} instance for each {es}
+node in the production cluster and use the default `scope: node`. When
```

Comment:
I find the `and the cluster does not use dedicated master-eligible nodes` a bit too hard/prescriptive. I would prefer to add a separate sentence that this is not recommended with dedicated master-eligible nodes and requires beefier master nodes than when using `scope:cluster`.

Reply:
ok, see faa0147